waterlog/src/icons/MapIcons.js

```diff
@@ -30,12 +30,12 @@ export const sensorFaultIcon = new L.icon({
   iconSize: new L.Point(25, 25)
 });
 
-export const sensorRedIcon = new L.icon({
-  iconUrl: require('../images/Red_sensor_dark.gif'),
-  iconAnchor: null,
+export const selectedComponentIcon = new L.icon({
+  iconUrl: require('../images/selectedComponent.png'),
```

Comment:
apply the same logic of icon request to this image

Reply:
done